config_reader.py

```diff
@@ -53,9 +53,15 @@ def get_fk_augmentation():
 def get_upstream_filters():
     return _config["upstream_filters"]
 
+def get_pre_constraint_sql():
+    return _config["pre_constraint_sql"] if "pre_constraint_sql" in _config else []
+
 def get_post_subset_sql():
     return _config["post_subset_sql"] if "post_subset_sql" in _config else []
 
+def get_max_rows_per_table():
+    return _config["max_rows_per_table"]
```

Comment:
`LIMIT ALL` is not valid in MySQL to my knowledge, and we don't want to force users to have to now provide a max_rows_per_table in order to get all rows. This should have some default behavior (i.e. return `None` if not defined in JSON), and we should explicit check for None in subset.py and only add the LIMIT clause when there's a non-None value here.